hummingbot/client/config/global_config_map.py

```diff
@@ -99,6 +99,14 @@ def using_wallet() -> bool:
                                                   prompt="Enter your Coinbase passphrase >>> ",
                                                   required_if=using_exchange("coinbase_pro"),
                                                   is_secure=True),
+    "bamboo_relay_use_coordinator":     ConfigVar(key="bamboo_relay_use_coordinator",
+                                                  prompt="Would you like to use the Bamboo Relay Coordinator >>> ",
+                                                  required_if=using_exchange("bamboo_relay"),
+                                                  is_secure=True),
+    "bamboo_relay_pre_emptive_soft_cancels":      ConfigVar(key="bamboo_relay_pre_emptive_soft_cancels",
```

Comment:
This looks like a yes/no question, in which case it's good to add `type_str` = "bool" there. Also the value does not have to be marked as `is_secure` (what that does is it masks the string when user types it in, which is useful for API keys)